templates/ros2_control/controller/test_dummy_chainable_controller.cpp

```diff
@@ -192,7 +195,7 @@ TEST_F(DummyClassNameTest, test_update_logic_fast)
 
   EXPECT_EQ(*(controller_->control_mode_.readFromRT()), control_mode_type::FAST);
   EXPECT_EQ(joint_command_values_[STATE_MY_ITFS], TEST_DISPLACEMENT);
-  EXPECT_TRUE(std::isnan((*(controller_->input_ref_.readFromRT()))->displacements[0]));
+  EXPECT_EQ((*(controller_->input_ref_.readFromRT()))->displacements[0], TEST_DISPLACEMENT);  
```

Comment:
i just adjusted this to work here, but wanted to get confirmation from you. I see that in mecanum we set reference msg to nan when ref_timeout = 0 and for too old msg (age_of_last_command > ref_timeout_)

in this particular test we have testing update fast logic where age_of_last_command <= ref_timeout_

and more over this test will be over written in the PR https://github.com/StoglRobotics/ros_team_workspace/pull/106
with a different name with similar logic test